src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -25,7 +30,8 @@ CREATE UNIQUE INDEX tab_disease_states_disease_state_uindex
 CREATE UNIQUE INDEX tab_disease_states_display_name_uindex
     on tab_disease_states (display_name);
 INSERT INTO tab_groups(group_name,display_name,description,parent_group,icon_name,visible,sequence,is_advanced,unique_id,pop_display_name,full_name) VALUES('DISEASE_STATES','Disease states','Disease states parameters',NULL,NULL,1,1,0,153,NULL,'Disease States');
-INSERT INTO tab_parameters(parameter_name,display_name,description,dimension,default_unit) VALUES('TargetGFR','TargetGFR TODO','TargetGFR TODO','Flow per body surface area',NULL);
+INSERT INTO tab_parameters(parameter_name,display_name,description,dimension,default_unit) VALUES('Target eGFR','Target eGFR','Target eGFR value','Flow per body surface area',NULL);

```

Comment:
@Yuri05 Based on a discussion with @prvmalik , I added this parameter as readonly so that the user can see the "Target value" that was entered.